File: Data Structures & Algorithms/longest-consecutive-sequence/submission-9.cs
public class Solution {
    public int LongestConsecutive(int[] nums) {
        HashSet<int> nums1 = new HashSet<int>(nums);
        int res=0;
        foreach( int x in nums1)
        {
            int curr = x;
            int count = 0;
            while(nums1.Contains(curr))
            {
                curr++;
                count++;
            }
            res = Math.Max(res,count);
        }
        return res;
        
    }
}

File: Data Structures & Algorithms/is-subsequence/submission-9.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        if len(s)>len(t):
            return False
        j=0
        i=0
        
        while j<len(t) and i<len(s):
            if s[i]==t[j]:
                i+=1
            j+=1

        return i==len(s)